src/sinol_make/commands/export/__init__.py

```diff
@@ -0,0 +1,287 @@
+import os
+import glob
+import shutil
+import tarfile
+import tempfile
+import argparse
+import yaml
+
+from sinol_make import util
+from sinol_make.helpers import package_util, parsers
+from sinol_make.commands.gen import gen_util
+from sinol_make.interfaces.BaseCommand import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Class for "export" command.
+    """
+
+    def get_name(self):
+        return "export"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Create archive for oioioi upload',
+            description='Creates archive ready to upload to oioioi. '
+                        'It is possible to create ocen and dlazaw debian packages for usage in contests '
+                        'with --ocen flag.'
+        )
+
+        parser.add_argument('-o', '--output', type=str, default='export',
+                            help='output directory. Default: export')
+        parser.add_argument('--ocen', action='store_true', default=False,
+                            help='create ocen (or dlazaw in case `dlazaw/` directory exists) debian package '
+                                 'for usage in contests')
+        parsers.add_compilation_arguments(parser)
+
+    def get_generated_tests(self):
+        """
+        Returns list of generated tests.
+        Executes ingen to check what tests are generated.
+        """
+        with tempfile.TemporaryDirectory() as tmpdir:
+            ingen_path = gen_util.get_ingen(self.task_id)
+            if os.path.splitext(ingen_path)[1] == '.sh':
+                ingen_exe = ingen_path
+            else:
+                ingen_exe = gen_util.compile_ingen(ingen_path, self.args, self.args.weak_compilation_flags)
+            if not gen_util.run_ingen(ingen_exe, tmpdir):
+                util.exit_with_error('Failed to run ingen.')
+
+            tests = glob.glob(os.path.join(tmpdir, f'{self.task_id}*.in'))
+            return [package_util.extract_test_id(test) for test in tests]
+
+    def copy_files(self, target_dir: str):
+        """
+        Copies files from current directory to target directory.
+        :param target_dir: Directory to copy files to.
+        """
+        files = ['config.yml', 'makefile.in', 'Makefile.in',
+                 'prog', 'doc', 'public', 'dlazaw']
```

Comment:
I found it when I was reading sio’s package processing code, files in this directory will be visible in “Files” tab in the contest. But I haven’t tested if this actually works

Reply:
Let's test it, just in case. Maybe we can just put everything into `public`, instead of `dlazaw`?

Reply:
`dlazaw` is also always uploaded to contestants' computers so that they can test it locally

Reply:
I tested it and the correct dir name is `attachments`, and this actually works

Reply:
So, can we fit everything into one directory, or must there be two separate ones?

Reply:
This would require task creators to zip `dlazaw` before placing it in the `attachments` directory. I don't think that this would save time for anyone and it would be bad for git.